packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -8,5 +8,6 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
-  "cancel_sfdx_command_text": "SFDX: Cancel Active Command"
+  "cancel_sfdx_command_text": "SFDX: Cancel Active Command",
+  "force_apex_class_create": "SFDX: Create Apex Class"
```

Comment:
So, I just realized after seeing the screenshot that @Sweetman sent that I hadn't actually reviewed this text for IDE 2, which seems to be in a `.java` file rather than in the messages and properties files that I looked at. Our standards say to avoid "Create a New...," because the "New" is implied. Please just use "Create an Apex Class" (or "Create a Visualforce Page", "Create a Visualforce Component", "Create a Lightning Component Bundle", etc.).